sigstore-java/src/main/java/dev/sigstore/encryption/Keys.java

```diff
@@ -74,7 +74,7 @@ public static PublicKey parsePublicKey(byte[] keyBytes)
       throw new InvalidKeySpecException("Invalid key, could not parse PEM section");
     }
     // special handling for PKCS1 (rsa) public key
-    if ((section == null) || (section.getContent() == null)) {
+    if ((section == null) || (section.getContent() == null) || (section.getContent().length == 0)) {
```

Comment:
Why do we explicitly verify for 0-byte long array?
What if the array is just 1 byte?